src/lib/GameList/GameListCarousel.svelte

```diff
@@ -0,0 +1,112 @@
+<script lang="ts">
+	import type { Game } from '$lib/games';
+	export let gamesData: Game[];
+	export let carouselTitle;
+
+	// export let game: Game;
+
+	// Target Group Functionality
+	// const targetGroupArr = game.targetGroup;
+	// targetGroupArr.sort(function (a, b) {
+	// 	return a - b;
+	// });
+	// const minGroup = targetGroupArr[0];
+	// const maxGroup = targetGroupArr.slice(-1)[0];
+	// const targetGroupString = `Groep ${minGroup} t/m ${maxGroup}`;
+</script>
+
+<section>
+	<h2>{carouselTitle} - {gamesData.length}</h2>
```

Comment:
```suggestion
	<h1>{carouselTitle} - {gamesData.length}</h1>
```

`Section` is sectioning content en vanaf dat punt kan je opneiuw beginnen met nummer

Reply:
Dit is dan ook consistent met de `/spellen` overzicht pagina